semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/callgraphextension/Method.scala

```diff
@@ -69,6 +69,6 @@ class Method(val wrapped: NodeSteps[nodes.Method]) extends AnyVal {
     * Outgoing call sites to methods where fullName matches `regex`.
     * */
   def callOutRegex(regex: String)(implicit callResolver: ICallResolver): NodeSteps[nodes.Call] =
```

Comment:
apply the same change for `callOutRegex`?

Reply:
I tried, but that leads to ambiguous implicit conversions :/